### DISCUSSION TITLE
Is there a way to do -A and -B without including the matched line in the output?

### DISCUSSION THREAD

**Discussion by Kommynct** [BurntSushi/ripgrep#2951] [Q&A] [answered]
I want the output to be the line above what i'm matching

rg -A 1 "Name:"

this does show the lines above, but also includes the lines with Name:

How can I achieve this?

**Comment by BurntSushi** [comment, marked as answer]
Yes. With shell pipelines: `rg -A 1 "Name:" | rg -v "Name:"`.
